core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/common/AbstractUserStoreManager.java

```diff
@@ -491,6 +490,22 @@ protected abstract void doUpdateRoleName(String roleName, String newRoleName)
     protected abstract String[] doListUsers(String filter, int maxItemLimit)
             throws UserStoreException;
 
+    /**
+     * Count claims.
+     *
+     * @param claimURI Claim uri
+     * @param valueFilter Filter
+     * @return null.
+     * @throws UserStoreException UserStoreException
+     */
+    public Long doCountClaims(String claimURI, String valueFilter) throws UserStoreException{
```

Comment:
doCountClaims - the name is not meaningful. I think it counts users based on the claim.